web/server/routers/account.ts

```diff
@@ -207,4 +213,13 @@ export const accountRouter = router({
         });
     }
   }),
+
+  search: procedure
+    .input(searchInput)
+    .query(async ({ input, ctx }) => {
+      const session = await Account.startSession();
+      const { searchSubject } = input;
+      const account = await searchAccount(searchSubject, session);
+      return account;
```

Comment:
Include a try catch that throws a TRPCError if there's an issue with the transactions. Also handle the case where `searchAccounts` returns null, which means there was a db issue